src/lib/stepper/step-header.ts

```diff
@@ -25,12 +33,15 @@ import {MatStepLabel} from './step-label';
   preserveWhitespaces: false,
 })
 export class MatStepHeader implements OnDestroy {
-  /** Icon for the given step. */
-  @Input() icon: string;
+  /** State of the given step. */
+  @Input() state: string;
```

Comment:
From the name I'm not sure that `state` is meant to be

Reply:
This component is internal anyway, but it's supposed to mean the state that the header is in (e.g. done, current).